### ISSUE TITLE
DeepMedic Install

### ISSUE THREAD

**Issue by JavierCoronel** [Kamnitsask/deepmedic#73]
Hi, I'm installing DeepMedic in another machine as usual, but I get this error:

```
Extracting deepmedic-0.6-py2.7.egg to /usr/local/lib/python2.7/dist-packages
deepmedic 0.6 is already the active version in easy-install.pth
Installing deepMedicRun script to /usr/local/bin
(So far, so good)

Installed /usr/local/lib/python2.7/dist-packages/deepmedic-0.6-py2.7.egg
Processing dependencies for deepmedic==0.6
Searching for numpy>=1.7.1
Reading http://pypi.python.org/simple/numpy/
Best match: numpy 1.13.0
Downloading https://pypi.python.org/packages/05/84/0feb999c05f252af50a5fbc463268044feda92cdaad8cb0d0a6073d76057/numpy-1.13.0.zip#md5=fd044f0b8079abeaf5e6d2e93b2c1d03
Processing numpy-1.13.0.zip
Running numpy-1.13.0/setup.py -q bdist_egg --dist-dir /tmp/easy_install-5CBMQN/numpy-1.13.0/egg-dist-tmp-VjVCIe
Running from numpy source directory.
/tmp/easy_install-5CBMQN/numpy-1.13.0/setup.py:367: UserWarning: Unrecognized setuptools command, proceeding with generating Cython sources and expanding templates
  run_build = parse_setuppy_commands()
Warning: Can't read registry to find the necessary compiler setting
Make sure that Python modules _winreg, win32api or win32con are installed.
/usr/lib/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
Warning: distutils distribution has been initialized, it may be too late to add a subpackage compatWarning: distutils distribution has been initialized... 

```

And then a lot of warnings...

It seems to be an error with numpy, but it's already installed and do not know what else could it be. Some help would be great.

Thanks,

Javier

**Comment by teamsar** [comment]
I think, the best thing to do is that you install pip first, and install all required packages using pip install --upgrade package_name_bla_bla_bla. Thanks

**Comment by JavierCoronel** [comment]
@teamsar thanks, you are right. 
That's how I installed numpy and scipy at first. I tried again, but still didn't work and the same error occured. It was on installing numpy 1.13 when obtaining from:
```
Downloading https://pypi.python.org/packages/05/84/0feb999c05f252af50a5fbc463268044feda92cdaad8cb0d0a6073d76057/numpy-1.13.0.zip#md5=fd044f0b8079abeaf5e6d2e93b2c1d03
Processing numpy-1.13.0.zip
```
So I decided to install that numpy version on my own with `wget ` that address and doing the same with scipy.
After that everything is working fine.

Javier
